src/test/java/com/alvarium/annotators/SourceCodeAnnotatorTest.java

```diff
@@ -0,0 +1,112 @@
+package com.alvarium.annotators;
```

Comment:
Needs copyright header. Please update the year to 2023 on all touched files in this PR.